tests/tst_decode.f

```diff
@@ -0,0 +1,121 @@
+!>
+!> This test is for decoding full grib2 message.
+!> Dusan Jovic, July, 2021
```

Comment:
Add your name and the date, and mention that this was derived from Dusan's program.

Reply:
Will add this docs.